clients/ordering/package.json

```diff
@@ -8,14 +8,18 @@
     "@material-ui/icons": "3.0.2",
     "@types/jest": "24.0.0",
     "@types/node": "10.12.21",
-    "@types/react": "16.8.7",
-    "@types/react-dom": "16.8.2",
+    "@types/react": "16.8.2",
```

Comment:
idk if it is smart to mess with this stuff unless Christian told you to. Hopefully, the versioning will work out fine.

Reply:
I don't think types should be an issue, but I would recommend keeping the same versions as the other clients.